pygtracker/pygtracker.py

```diff
@@ -12,10 +12,39 @@ def __init__(self):
         pass
 
     # Start: Register the courses
+    def register_courses(file):
+      """Read and store the input csv file as a pandas dataframe.
```

Comment:
```suggestion
      """
      Read and store the input csv file as a pandas dataframe.
```

Not sure if putting the text right after """ will affect the docstring displayed?